pyproject.toml

```diff
@@ -10,6 +10,7 @@ python = "^3.9"
 cairo-lang = "^0.10.1"
 openzeppelin-cairo-contracts = "0.5.0"
 amarna = "^0.1.5"
+python-dotenv = "^0.21.0"
```

Comment:
quite surprised that poetry did this while this deps is also in `dev-deps` below. I think it's more a dev deps than a kakarot deps though so I would go for not updating this file